package.json

```diff
@@ -51,8 +53,12 @@
 	"dependencies": {
 		"@emotion/react": "^11.4.1",
 		"@emotion/styled": "^11.3.0",
+		"@mui/base": "5.0.0-alpha.89",
 		"@mui/icons-material": "^5.0.0",
 		"@mui/material": "^5.0.0",
+		"@mui/system": "^5.9.0",
+		"@mui/types": "^7.1.4",
+		"@mui/utils": "^5.9.0",
```

Comment:
idk how this was building before, you were missing these dependencies. the repo errored for me without these